x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial-grid.csv-spec

```diff
@@ -2,27 +2,43 @@
 # Tests for geo_grid function: ST_GEOHASH
 ###############################################
 
-geohashStringToLong
-required_capability: spatial_grid
+toGeohash
+required_capability: spatial_grid_types
 
-// tag::geohash_to_long[]
-ROW geohash = "u3bu"
-| EVAL geohashLong = ST_GEOHASH_TO_LONG(geohash)
-// end::geohash_to_long[]
+// tag::to_geohash[]
+ROW string = "u3bu"
+| EVAL geohash = TO_GEOHASH(string)
+// end::to_geohash[]
 ;
 
-// tag::geohash_to_long-result[]
-geohash:keyword | geohashLong:long
-u3bu            | 13686180
-// end::geohash_to_long-result[]
+// tag::to_geohash-result[]
+string:keyword | geohash:geohash
+u3bu           | u3bu
+// end::to_geohash-result[]
+;
+
+toGeohashToLong
+required_capability: spatial_grid_types
+
+// tag::to_geohash_to_long[]
+ROW string = "u3bu"
+| EVAL geohash = TO_GEOHASH(string)
+| EVAL geohashLong = TO_LONG(geohash)
+// end::to_geohash_to_long[]
+;
+
+// tag::to_geohash_to_long-result[]
+string:keyword | geohash:geohash | geohashLong:long
+u3bu           | u3bu            | 13686180
```

Comment:
These test expectations suggest that `geohash` is a type that users will get as return value and can see in the query output when running the query.

However, trying locally with `ROW geohash = (13686180::long)::geohash | EVAL geohashString = TO_STRING(geohash)` I get a `null`-filled column of type `unsupported`.

I assume the grid types aren't supposed to be non-representable? That is, internal types like `DATE_PERIOD` and `TIME_DURATION`? The latter cannot make it into the output of a query.